src/accordion/Accordion.test.html

```diff
@@ -0,0 +1,29 @@
+`
+<div id="accordion">
```

Comment:
@victorwhy Good question! Ideal situation is actually to render a Nunjucks template on the fly where we use the `accordion` macro. This way, we'll always be in sync with the latest updates to the HTML structure if the macro ever changes. Also, if the template call resembles a real macro call, then it will be testing exactly the HTML that's being used on production.

The only reason I'm not using the template rendering approach at the moment is because I ran into some issues with Nunjucks file pathing, which Claudina is going to help me with later.